ConnectionToBioEmbeddings.md

```diff
@@ -0,0 +1,83 @@
+# Schema for the connection of FLIP with bio-embeddings (via biotrainer)
+
+Proposed tree of folders:
+
+```
+FLIP
+│   ...    
+│
+└───baselines/
+│
+└───collect_splots/
+│
+└───data/ (not in GitHub)
+│
+└───helpers/
+│
+└───splits/
+│
+└───advanced/ <---------------------- Here goes the connection to bio-embeddings (Name for the folder?)
+│   │
+│   └───advenced_environment.yml <-- File with the required environment for the execution 
+│   │
+│   └───train_and_eval.py <--------- Here goes the script for the training and evaluation of the splits
+│   │
+│   └───utils.py <------------------- Here go auxiliary functions (if required)
+│   │   
+│   └───configs_bank/ <-------------- Here go the configuration files (recommended config) for the different splits
+│   │   
+│   │   config_aav.txt
+│   │   config_gb1.txt
+│   │   config_meltome.txt
+│   │   ...
+│   │
+```
+
+
+## About `train_and_eval.py` file:
+
+### Arguments
+- `split`: name of the split to be evaluated from a list of available splits. Used to obtain tmp files `sequence_file` and `labels_file`, and database of origin used to select the corresponding config file.
```

Comment:
This will largely depend on the data standardization that we choose for biotrainer. Maybe it will be enough to provide the split and use the file(s) provided there :)